code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -247,11 +247,6 @@
 /obj/machinery/chem_master/attack_ai_secondary(mob/user, list/modifiers)
 	return attack_hand_secondary(user, modifiers)
 
-/obj/machinery/chem_master/ui_assets(mob/user)
-	return list(
-		get_asset_datum(/datum/asset/spritesheet/chemmaster)
```

Comment:
Is this spritesheet used elsewhere? If not, should we delete it?

Reply:
pill press